Makefile

```diff
@@ -401,46 +401,10 @@ docker_clean_volumes:
 .PHONY: docker_cv # Docker clean volumes
 docker_cv: docker_clean_volumes
 
-
-.PHONY: docs_no_links # Build docs
-docs_no_links: clean_docs check_docs_dollars
-	@# Rebuild the index from README.md, to have in the home of Sphinx a copy of README.md
-	echo "  .. Warning, auto-generated by \`make docs\`, don\'t edit" > docs/index.rst
-	echo "" >> docs/index.rst
-	pandoc --from markdown --to rst docs/README.md >> docs/index.rst
-	@# To be sure there is a blank line at the end
-	echo "" >> docs/index.rst
-	cat docs/index.toc.txt >> docs/index.rst
-	@# Create _static if nothing is commited in it
-	mkdir -p docs/_static/
-	@# Generate the auto summary of documentations
-	@# Cannot do without specifying top module currently with sphinx-apidoc
-	poetry run sphinx-apidoc --implicit-namespaces -o docs/_apidoc $(CONCRETE_PACKAGE_PATH)
-	@# Doing a copy of docs, where we modify files
-	rm -rf docs-copy
-	cp -r docs docs-copy
-	@# Admonitions
-	./script/make_utils/sphinx_gitbook_admonitions.sh --gitbook_to_sphinx
-	@# Check that there is no more GitBook hint
-	! grep -r "hint style" docs-copy
-	@# Replace $$, $/$ and /$$ by $
-	./script/make_utils/fix_double_dollars_issues_with_mdformat.sh docs-copy --single_dollar
-	@# Replace `href="*.md` patterns with `href="*.html` because Sphinx does not handle them
-	./script/make_utils/fix_md_to_html_conversion_from_sphinx_in_href.sh docs-copy
-	@# Replace `references/api/README.md` with `_apidoc/modules.html`.
-	./script/make_utils/fix_api_readme_reference.sh docs-copy
-	@# Fix not-compatible paths
-	./script/make_utils/fix_gitbook_paths.sh docs-copy
-	@# Docs
-	cd docs-copy && poetry run "$(MAKE)" html SPHINXOPTS='-W --keep-going'
-	@# Copy images from GitBook
-	cp docs/.gitbook/assets/*.png docs-copy/_build/html/_images
-	cp -r use_case_examples docs-copy/_build/
-	cp -r docs-copy/_build docs/
-	rm -rf docs-copy
-
 .PHONY: docs # Build docs and check links
-docs: docs_no_links
```

Comment:
could you keep `docs_no_links` target ? so that `make docs` is `docs_no_links` + `check_links` like before ? `docs_no_links` is used in the CI in order to avoid checking links when releasing, as it is sometimes flaky (and independent from us)

Reply:
done